extensions/wikia/WikiFactory/Close/SpecialCloseWiki_body.php

```diff
@@ -230,8 +230,10 @@ private function doProcess() {
 	}
 
 	private function prefixMainDomain( $cityId ) {
-		$mainDomain = substr( WikiFactory::getVarValueByName("wgServer", $cityId), 7 );
-		if(!empty($mainDomain)) {
+		//local prefix to not accidentally mix it with global wgServer
+		$localWgServer = WikiFactory::getVarValueByName( "wgServer", $cityId );
+		if ( preg_match( "^https?:\/\/", $localWgServer, $matches ) ) {
+			$mainDomain = $matches[1];
```

Comment:
The same problem with regex. Please consider having that logic in one place instead of duplicating problems ;)